platform/core/src/DICOMSR/dataExchange.js

```diff
@@ -78,4 +82,112 @@ const storeMeasurements = async (measurementData, filter, server) => {
   }
 };
 
-export { retrieveMeasurements, storeMeasurements };
+const downloadReport = measurementData => {
+  const srDataset = generateReport(measurementData);
+  const reportBlob = dcmjs.data.datasetToBlob(srDataset);
+
+  //Create a URL for the binary.
+  var objectUrl = URL.createObjectURL(reportBlob);
+  window.location.assign(objectUrl);
+};
+
+const generateReport = measurementData => {
+  const ids = measurementData.map(md => md.id);
+  const filteredToolState = _getFilteredCornerstoneToolState(ids);
+
+  const report = MeasurementReport.generateReport(
+    filteredToolState,
+    cornerstone.metaData
+  );
+
+  return report.dataset;
+};
+
+const storeMeasurements = async (measurementData, dataSource) => {
```

Comment:
Same here. If this takes a `dataSource`, can we call out the required props in JSDoc?

`@param {func} dataSource.store.dicom - does stuff`